accounts/auth.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from django.contrib.auth.backends import ModelBackend, UserModel
+
+logger = logging.getLogger(__name__)
+
+
+class EmailAsUsernameModelBackend(ModelBackend):
+    def authenticate(self, request, username=None, password=None, **kwargs):
+        if username and "@" in username:
+            try:
+                user = UserModel._default_manager.get(email=username)
```

Comment:
```suggestion
                user = UserModel._default_manager.get(email__iexact=username)
```

Per RFC e-mail addresses are case insensitive, so has to be our search.